polling_stations/urls.py

```diff
@@ -69,6 +69,7 @@
     url(r"^feedback/", include("feedback.urls")),
     url(r"^report_problem/", include("bug_reports.urls")),
     url(r"^league_table/", include("data_collection.urls")),
+    url(r"^dashboard/", include("dashboard.urls", namespace="dashboard")),
```

Comment:
I think I preferred your intial approach of only mouting this route `if settings.DEBUG`. I'm quite keen to not accidentally deploy this. In fact, it would also be quite nice to do

```py
if DEBUG:
    INSTALLED_APPS.append("dashboard")
```

in the settings file too, just for belt + brances.

Reply:
Done, and it's below the `.local` import, so it can be disabled from there.